src/batch-client/index.ts

```diff
@@ -796,7 +797,13 @@ export class ZimbraBatchClient {
 				},
 				...rest
 			}
-		}).then(res => res.cn.map((contact: any) => normalize(Contact)(contact)));
+		}).then(res =>
+			res.cn.map((cn: any) => {
+				const contact = castNicknameAsString(cn);
```

Comment:
all other contact related functions are calling normalizeOtherAttr, why getContact is not doing it? ideally it should also do same as others

Reply:
ok checking i didn’t updated the code